src/graphlog.rs

```diff
@@ -127,10 +126,9 @@ mod sapling {
 }
 
 pub fn get_graphlog<'a, K: Clone + Eq + Hash + 'a>(
-    #[allow(unused)] settings: &UserSettings,
+    settings: &UserSettings,
     formatter: &'a mut dyn Write,
 ) -> Box<dyn GraphLog<K> + 'a> {
-    #[cfg(feature = "sapling")]
     {
         use renderdag::GraphRowRenderer;
         use sapling::SaplingGraphLog;
```

Comment:
Move these `use` directives to the top of the file now?

Reply:
Done.